static/src/stylesheets/module/commercial/gimbap/_gimbap-richmedia.scss

```diff
@@ -0,0 +1,72 @@
+%gimbap-zindex {
+    position: relative;
+    z-index: 1;
+}
+
+.gimbap-richmedia__title {
+    @extend %gimbap-zindex;
+    @include fs-headline(3);
+
+    font-weight: bold;
+    line-height: .8;
+    color: #ffffff;
+
+    .has-flex & {
```

Comment:
Why? You don't need this guard since browsers which don't support flexbox won't understand `align-self`. Plus this way it works without JS 💥